test/helpers/db_helper.rb

```diff
@@ -108,7 +108,7 @@ def seed_simple_database_with_single_table
     # Create some holes in the data.
     delete_statement = source_db.prepare("DELETE FROM #{full_table_name(DEFAULT_DB, DEFAULT_TABLE)} WHERE id = ?")
     140.times do
-      delete_statement.execute(Random.rand(max_id) + 1)
+      delete_statement.execute(Random.rand(max_id))
```

Comment:
Since we report progress at the end of the run, I want to check the LastSuccessfulPK and TargetPK in tests. This requires the last id to be predictable.